packages/extension_google_sign_in_as_googleapis_auth/example/pubspec.yaml

```diff
@@ -4,8 +4,13 @@ publish_to: none
 
 environment:
   sdk: ">=2.12.0 <3.0.0"
-  flutter: ">=1.20.0"
```

Comment:
Let's re-add this.

Reply:
Will do, I thought 1.20 was already "very old", but it won't hurt to have this check here.